frontend/src/App.tsx

```diff
@@ -39,6 +40,15 @@ export default function App() {
         null
     );
 
+    const [leagueParams, setLeagueParams] = useState<LeagueParams>({
+        league: "GeneralLeague",
+        tier: "tier1",
+        year: new Date().getFullYear(),
+    });
```

Comment:
This is what I used as a stand-in for query params - will adjust as needed to match your implementation!

Reply:
Awesome! You can check my PR to get the exact types, should be pretty straightforward.

Reply:
Sweet mostly got it - updated the tier values to be capitalized now